app/controllers/admin/document_collection_groups_controller.rb

```diff
@@ -5,7 +5,7 @@ class Admin::DocumentCollectionGroupsController < Admin::BaseController
   def index
     @groups = @collection.groups.includes(
       memberships: [
-        { document: { latest_edition: %i[organisations translations] } },
+        { document: { latest_edition: %i[translations] } },
```

Comment:
> The fix is to not use organisations in the includes query, necessitating an additional query for organisations.

Should there be another query here, or does that happen elsewhere?

Reply:
It happens automatically elsewhere when `.organisations` is called.